server.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const express = require('express');
+
+// Constants
+const PORT = 3000;
+
+// App
+const app = express();
+
+app.use(express.static('public'));
+
+app.get('/', function (req, res) {
+   res.sendFile(__dirname + "/" + "index.html");
```

Comment:
there are a number of ways of doing this (using routes, `res.render`, etc.) Couldn't tell if there is a clear best practice. Opinions?